requirements.txt

```diff
@@ -1,3 +1 @@
-# Only for tests on Python < 2.7:
-argparse
-unittest2
+virtualenv==13.1.2
```

Comment:
I don't think that belongs here. If our tools are using too old a virtualenv, we must fix them.